File: HackerRank/Python/Medium/M0003.py
# Problem Statement: https://www.hackerrank.com/challenges/python-time-delta/problem

from datetime import datetime

for _ in range(int(input())):
	t1, t2 = (datetime.strptime(input(), "%a %d %b %Y %H:%M:%S %z") for _ in range(2))
	sec = int(abs((t1 - t2).total_seconds()))
	print(sec)

File: HackerRank/C/Medium/M0003.c
/*
Problem Statement: https://www.hackerrank.com/challenges/reverse-array-c/problem
*/

#include <stdio.h>

void swap(int* a, int* b) {
	int temp = *a;
	*a = *b;
	*b = temp;
}

int main() {
	int n;
	scanf("%d", &n);
	int arr[n];
	
	for (int i = 0; i < n; i++)
		scanf("%d", arr + i);
	for (int i = 0; i < n >> 1; i++)
		swap(&arr[i], &arr[n - i - 1]);
	for (int i = 0; i < n; i++)
		printf("%d ", arr[i]);

	return 0;
}